src/Symfony/Component/TypeInfo/Type/UnionType.php

```diff
@@ -57,4 +57,32 @@ public function __toString(): string
 
         return $string;
     }
+
+    /**
+     * BC Layer for Symfony\Component\PropertyInfo\Type.
+     *
+     * @internal
```

Comment:
can the comment hint future maintainers about when this can be removed? (same in all similar situations)